### PR TITLE
Upgrade admin and account console to PatternFly 5

### LINKED ISSUE #21344: Upgrade account theme to PatternFly 5
The account theme will have to be upgraded to the latest version of PatternFly.

### LINKED ISSUE #21345: Upgrade admin theme to PatternFly 5
The admin theme will have to be upgraded to the latest version of PatternFly. Jenny is already testing PatternFly 5 with admin console (see #21303).

### PR DESCRIPTION
**Pull request by jonkoops**
Closes #21345
Closes #21344

### CODE REVIEW

File: js/apps/account-ui/src/account-security/SigningIn.tsx [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+	Spinner,
+	Split,
+	SplitItem,
+	Title
+} from '@patternfly/react-core';
+import {
+	Dropdown,
+	DropdownItem,
+	KebabToggle
+} from '@patternfly/react-core/deprecated';
```
Comment (jonkoops) [inline js/apps/account-ui/src/account-security/SigningIn.tsx:19]: I've [logged an issue](https://github.com/keycloak/keycloak/issues/28197) to clean up these deprecated components later.

**Review by edewit — DISMISSED** [review]

**Review by hmlnarik — DISMISSED** [review]
Great job everyone!

**Review by jonkoops — COMMENTED** [review]
LGTM

**Review by hmlnarik — APPROVED** [review]

**Review by edewit — APPROVED** [review]

**Review by mfrances17 — APPROVED** [review]
LGTM 👍

**Review by agagancarczyk — APPROVED** [review]

**Pull request merged**